design/continuous-delivery.md

```diff
@@ -0,0 +1,329 @@
+# Continuous Delivery for CDK Apps
+
+This is a specification for continuous delivery support for CDK apps. It
```

Comment:
YEASSSSSSSS. I'm so psyched by this project - thanks for putting this together!

Reply:
🎉